core/ErrorFlusherStdout.h

```diff
@@ -17,7 +17,8 @@ class ErrorFlusherStdout : public ErrorFlusher {
     ErrorFlusherStdout() = default;
     ~ErrorFlusherStdout() = default;
 
-    void flushErrors(spdlog::logger &logger, std::vector<std::unique_ptr<ErrorQueueMessage>> error) override;
+    void flushErrors(spdlog::logger &logger, std::vector<std::unique_ptr<ErrorQueueMessage>> errors,
+                     const GlobalState &gs, core::FileRef file) override;
     void flushErrorCount(spdlog::logger &logger, int count) override;
```

Comment:
`flushErrorCount` and `flushAutocorrects` are only relevant on the CLI. Can we remove them from the interface and `ErrorQueue`, and have `realmain` call them directly on `ErrorFlusherStdout`?

Reply:
(`ErrorFlusherStdout` can simply count the number of errors it prints, so we can remove the atomic error counter from `ErrorQueue`)